htdocs/compta/bank/class/paymentvarious.class.php

```diff
@@ -125,6 +125,16 @@ class PaymentVarious extends CommonObject
 	 */
 	public $fk_user_modif;
 
+	/**
+	 * @var int ID
+	 */
+	public $user_modif;
```

Comment:
We already have 
$user_modification_id
into commonobject. We should instead rename user_modif into user_modification_id